input/index.html.in

```diff
@@ -19,7 +19,7 @@ extra_headers: <link rel="stylesheet" href="{{site_url}}/index.css" type="text/c
           secure communication across Kubernetes clusters with no VPNs
           or special firewall rules.</p>
 
-        <p>With Skupper, your application can span multiple cloud
+        <p>With Skupper, your application can securely span multiple cloud
```

Comment:
The previous sentence says "secure communication", so I feel like this point is already made in this context.   The point is then made a second time shortly after, in the "Security is built in" box.